### PR TITLE
Enable internal communication on mine record

### PR DESCRIPTION
**Pull request by mmumby**
# Main
- Exposed 'Internal Communication' in Prod
- Updated GET to only return comments less than a year
- Added some disclaimers (see pictures)
Screen Shot 2021-04-09 at 4 00 17 PM
Screen Shot 2021-04-09 at 4 03 06 PM
Screen Shot 2021-04-09 at 3 59 16 PM

## Other
- Added a truncateFileName function to the link to CRTS on the consultation tab

### CODE REVIEW

File: services/core-web/src/components/mine/MineComments.js [hunk 1]
```diff
@@ -46,8 +46,12 @@ export class MineComments extends Component {
   }
 
   render() {
+    console.log(this.props.comments);
```
Comment (exzizt) [inline services/core-web/src/components/mine/MineComments.js:49]: Remove debug

File: services/core-api/app/api/mines/comments/resources/mine_comment.py [hunk 2]
```diff
@@ -31,10 +31,12 @@ def get(self, mine_guid):
             raise NotFound('Mine not found')
 
         current_app.logger.info(f'Retrieving comments for {mine}')
+        
+        year = datetime.now(timezone.utc) - timedelta(days=365)
 
-        comments = [comment.__dict__ for comment in mine.comments]
+        new_comments = [x for x in mine.comments if x.comment_datetime >= year]
```
Comment (exzizt) [inline services/core-api/app/api/mines/comments/resources/mine_comment.py:37]: Nice :)

**Review by exzizt — APPROVED** [review]
Looks good ✔️

File: services/core-web/src/components/mine/MineComments.js (hunk 1 above)
Comment (mmumby) [reply services/core-web/src/components/mine/MineComments.js]: removed!

**Review by exzizt — APPROVED** [review]

**Pull request merged**